kpatch/kpatch

```diff
@@ -349,35 +352,44 @@ disable_patch () {
 			[[ -z "$out" ]] && break
 			echo "$out" 1>&2
 			if [[ ! "$out" =~ "Device or resource busy" ]]; then
-				die "failed to disable module $modname"
+				return 1
 			fi
 
 			# "Device or resource busy" means the activeness safety check
 			# failed.  Retry in a few seconds.
 			i=$((i+1))
 			if [[ $i -eq $MAX_LOAD_ATTEMPTS ]]; then
-				die "failed to disable module $modname"
+				return 1
 			else
 				warn "retrying..."
 				sleep $RETRY_INTERVAL
 			fi
 		done
 	fi
+}
+
+disable_patch_strict () {
+	local modname=$1
```

Comment:
nit: file convention seems to be `local modname="$1"`

Reply:
Fixed in new version.